examples/nodejs/cache/doc-example-files/config-and-error-handling.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ *
+ * This file contains examples of consuming the JavaScript APIs, for use as examples
+ * in the public dev docs.  Each function name that begins with `example_` is available
+ * to the dev docs to inject into the code snippets widget for the specified API.
+ *
+ * These examples should all be JavaScript; we can add TypeScript-specific examples in
+ * a second file in the future if desired.
+ *
+ */
+import {
+  CacheClient,
+  Configurations,
+  CredentialProvider,
+  CreateCache,
+  DeleteCache,
+  CacheFlush,
+  ListCaches,
+  CacheSet,
+  CacheGet,
+  CacheDelete,
+  AuthClient,
+  AllDataReadWrite,
+  ExpiresIn,
+  TokenScopes,
+  DisposableTokenScopes,
+  AllCaches,
+  AllTopics,
+  CacheRole,
+  TopicRole,
+  GenerateApiKey,
+  RefreshApiKey,
+  GenerateDisposableToken,
+  CacheIncrement,
+  CacheItemGetType,
+  CacheSetIfNotExists,
+  CacheListFetch,
+  CacheListConcatenateBack,
+  CacheListConcatenateFront,
+  CacheListLength,
+  CacheListPopBack,
+  CacheListPopFront,
+  CacheListPushBack,
+  CacheListPushFront,
+  CacheListRemoveValue,
+  CacheListRetain,
+  ItemType,
+  CacheDictionaryFetch,
+  CacheDictionaryGetField,
+  CacheDictionaryGetFields,
+  CacheDictionarySetField,
+  CacheDictionarySetFields,
+  CacheDictionaryIncrement,
+  CacheDictionaryRemoveField,
+  CacheDictionaryRemoveFields,
+  CacheSetAddElement,
+  CacheSetAddElements,
+  CacheSetFetch,
+  CacheSetRemoveElement,
+  CacheSetRemoveElements,
+  CacheSortedSetPutElement,
+  CacheSortedSetPutElements,
+  CacheSortedSetFetch,
+  CacheSortedSetGetRank,
+  CacheSortedSetGetScore,
+  CacheSortedSetGetScores,
+  CacheSortedSetIncrementScore,
+  CacheSortedSetRemoveElement,
+  CacheSortedSetRemoveElements,
+  TopicClient,
+  TopicPublish,
+  TopicSubscribe,
+  TopicItem,
+  TopicConfigurations,
+  PreviewLeaderboardClient,
+  LeaderboardConfigurations,
+  ILeaderboard,
+  LeaderboardDelete,
+  LeaderboardFetch,
+  LeaderboardLength,
+  LeaderboardOrder,
+  LeaderboardRemoveElements,
+  LeaderboardUpsert,
+  ListWebhooks,
+  DeleteWebhook,
+  PutWebhook,
+  RotateWebhookSecret,
+  GetWebhookSecret,
+  SdkError,
+  MomentoErrorCode,
+} from '@gomomento/sdk';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+async function configuration_ConstructWithNoConfig() {
```

Comment:
@allenheltondev everything in this file is intended to show the difference between explicitly passing in a Configuration vs. using the default, plus how to use the ThrowOnErrors stuff. I plan to pull these snippets into code blocks in the dev docs in another PR momentarily so if there is anything in here you don't like lmk.